src/pages/SettingsPage.js

```diff
@@ -52,14 +59,28 @@ export default function SettingsPage({
     setAllPads(sortedPads);
     setStoragedPadSettings(sortedPads);
   }
+  function samplePreview() {
+    const player = new Tone.Player(selectedSample).toDestination();
+    Tone.loaded().then(() => {
+      player.start();
+    });
+  }
 }
 
 const SettingsContainer = styled.div`
   display: flex;
   flex-direction: column;
   flex-wrap: wrap;
 `;
-
+const HeadingContainer = styled.div`
```

Comment:
I think it would be more semantically to use a `header`element? Please check whether this would work as well instead of the `div` (and let me know 🤓).

Reply:
Yes you are right. Maybe this is better 😅👍🏻